.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -178,7 +178,7 @@ fun applyDefaults(model: CIBuildModel, buildType: BaseGradleBuildType, gradleTas
                 executionMode = BuildStep.ExecutionMode.ALWAYS
                 tasks = "tagBuild"
                 gradleParams = "$gradleParameterString -PteamCityUsername=%teamcity.username.restbot% -PteamCityPassword=%teamcity.password.restbot% -PteamCityBuildId=%teamcity.build.id% -PgithubToken=%github.ci.oauth.token%"
-                buildFile = "gradle/buildTagging.gradle"
+                buildFile = "build.gradle.kts"
```

Comment:
just remove this line?

Reply:
Well I don't quite understand, IIUC, `tagBuild` task is registered by `org.gradle.ci.tagging.plugin.TagSingleBuildPlugin`. How does it work to run `./gradlew tagBuild` without `buildTagging.gradle`?

Reply:
@blindpirate We apply the build scan plugin to the main build, too: https://github.com/gradle/gradle/blob/release/build.gradle.kts#L39

I waited for some time for this follow up change so all branches have the change.

Reply:
@big-guy I removed the line.